### PR TITLE
#2301 - Feature/join org us11.1

### LINKED ISSUE #2301: User Story 11.1: Notification Settings for New Applicant

### PR DESCRIPTION
**Pull request by ChunPingYang**
### What does this pull request aim to achieve? 💯
Add New Applicant checkbox in Notification

_Please be concise and link any related issue(s):_

- [V ] I have checked that no one else is working on similar changes.
- [V ] I have read the latest [**README**](README.md) and understand the development workflow.
- [V] The name of this branch is: **`feature/<branch_name>`** for a regular feature, **`hotfix/<branch_name>`** for a hotfix, and **`service/<branch_name>`** for a microservice, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [V] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [V] There are no merge conflicts.
- [V] There are no console warnings and errors.
- [V] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [V] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/components/Input/NotifyPreferenceInput.js [hunk 1]
```diff
@@ -80,7 +81,7 @@ const NotifyPreferenceInput = ({
       <WhiteSpace />
       <HelpWrapper>
         {Object.entries(NotifyGroup).map(([key1, label1]) => (
-          <>
+          <div key={key1}>
```
Comment (aditilamkhede) [inline client/src/components/Input/NotifyPreferenceInput.js:91]: What is this code doing? Also the checkbox for new applicant should only be visible to org owner or admin.

**Review by aditilamkhede — CHANGES_REQUESTED** [review]

File: client/src/components/Input/NotifyPreferenceInput.js (hunk 1 above)
Comment (ChunPingYang) [reply client/src/components/Input/NotifyPreferenceInput.js:91]: 1. I found that under the iteration of this collection, line83 does not have the key, so there is an error which occurs saying need key, therefore I add key right there. #
2. About the checkbox, actually I found there are two files "EditNotification.js" and "EditOrganisationNotification.js" have almost the same code, what is the difference between both of them? How do I check the user is (org owner or admin) or not?

**Comment by ChunPingYang** [comment]
@aditilamkhede I already fix the issue, please review it.

**Comment by aditilamkhede** [comment]
> @aditilamkhede I already fix the issue, please review it.

@ChunPingYang I saw the New applicant checkbox in Individual user profile and it is not visible in Organization profile.

It should be implemented vice a versa.

**Comment by ChunPingYang** [comment]
> > @aditilamkhede I already fix the issue, please review it.
> 
> @ChunPingYang I saw the New applicant checkbox in Individual user profile and it is not visible in Organization profile.
> 
> It should be implemented vice a versa.

**Comment by ChunPingYang** [comment]
> > > @aditilamkhede I already fix the issue, please review it.
> > 
> > 
> > @ChunPingYang I saw the New applicant checkbox in Individual user profile and it is not visible in Organization profile.
> > It should be implemented vice a versa.

@aditilamkhede 
How do you test my code ? I run my code and show the view as what I expected. 
Screen Shot 2021-04-01 at 1 23 16 PM
Screen Shot 2021-04-01 at 1 24 07 PM

**Comment by aditilamkhede** [comment]
> > > > @aditilamkhede I already fix the issue, please review it.
> > > 
> > > 
> > > @ChunPingYang I saw the New applicant checkbox in Individual user profile and it is not visible in Organization profile.
> > > It should be implemented vice a versa.
> 
> @aditilamkhede
> How do you test my code ? I run my code and show the view as what I expected.
> Screen Shot 2021-04-01 at 1 23 16 PM
> Screen Shot 2021-04-01 at 1 24 07 PM

@ChunPingYang your branch has conflict, please resolve those conflicts. then we might have to redeploy your changes.

**Comment by hiran11** [comment]
@ChunPingYang  When i try to switch account in help board page between individual account and organisation account an error message is populated. 

image

**Comment by aditilamkhede** [comment]
@ChunPingYang looks fixed but changing the value of new applicant doesn't reflect newly changed value.

**Comment by AndersonNY** [comment]
Screen Shot 2021-04-05 at 9 19 24 AM I ONLY SEE THE NEW APPLICANT BOX on orgn page.

**Comment by AndersonNY** [comment]
As a requirement "GIVEN the org admin is in the ‘Notifications’ page/screen WHEN they toggle on the ‘New Applicant’ THEN they must receive email notifications whenever a new application comes through."  Yes the new applicant button is working as a expected but email notifications is not working whenever a new application comes trough. I tried that a few times. 

Screen Shot 2021-04-14 at 3 33 17 AM

**Comment by xJamesScott** [comment]
temporarily closed for troubleshooting

**Pull request closed without merging**